src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/View/Controller/ExposureSubmissionQRInfoViewController.swift

```diff
@@ -76,4 +83,16 @@ class ExposureSubmissionQRInfoViewController: DynamicTableViewController, Footer
 		dynamicTableViewModel = viewModel.dynamicTableViewModel
 		tableView.separatorStyle = .none
 	}
+	
+	private func setupNavigationBar() {
+		// set the bar tint to white
+		parent?.navigationController?.navigationBar.tintColor = .white
+
+		// create a transparent navigation bar
+		let emptyImage = UIImage()
+		navigationController?.navigationBar.setBackgroundImage(emptyImage, for: .default)
+		navigationController?.navigationBar.shadowImage = emptyImage
+		navigationController?.navigationBar.isTranslucent = true
+		navigationController?.view.backgroundColor = .clear
+	}
 }
```

Comment:
Setting an empty image will break the navigationBar when pop or push to another screen.
I Guess you have to store the old values somewhere and reset them on push or pop operations

Reply:
This is how it is done in AntigenTestProfileViewController line, just wondering why would it break to set that up again?